Ollivanders/src/net/pottercraft/Ollivanders2/Potion/IngredientType.java

```diff
@@ -7,35 +7,51 @@
 
 public enum IngredientType
 {
+   ACONITE (Material.SPIDER_EYE, (short)0, "Aconite"),
```

Comment:
Aconite is a plant so we would want this ingredient to look like a plant. Based on the picture on wikia - http://harrypotter.wikia.com/wiki/Aconite - it looks like Allium or Lilac looks most like it.

Reply:
I wasnt able to find any other flower types when assigning a (Material.X) , is it possible to add them? Or is that something that is client side?

Reply:
You have to use the second arg on the enum - the short - to get different flower types.

Allium:
ACONITE (Material.RED_ROSE, (short)2, "Aconite"), //allium

Lilac:
ACONITE (Material.DOUBLE_PLANT, (short)1, "Aconite"), //lilac

Reply:
You have to use the second arg on the enum - the short - to get different flower types.

Allium:
ACONITE (Material.RED_ROSE, (short)2, "Aconite"), //allium

Lilac:
ACONITE (Material.DOUBLE_PLANT, (short)1, "Aconite"), //lilac

Reply:
I made this allium for now.